OpenSim/Tools/Test/sphere.vtp

```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0"?>
+<VTKFile type="PolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
```

Comment:
Agreed, and that change has been made to use non-uniformly scaled block.vtp. Screen capture from 3.3 and master have been added as well. Thanks @aseth1